src/ergw_aaa_diameter.erl

```diff
@@ -10,8 +10,7 @@
 -compile({parse_transform, cut}).
 
 %% API
--export([validate_function/1,
-	 initialize_function/2]).
+-export([validate_function/1, validate_handler/1, initialize_function/2]).
```

Comment:
ergw_aaa_diameter is not a handler, so adding that here make no sense.

Reply:
Removed. Done.